telegram/_passport/passportfile.py

```diff
@@ -111,6 +111,9 @@ def de_json_decrypted(
         if not data:
             return None
 
+        loc_tzinfo = extract_tzinfo_from_defaults(bot)
+
+        data["file_date"] = from_timestamp(data.get("last_error_date"), tzinfo=loc_tzinfo)
```

Comment:
```suggestion
        data["file_date"] = from_timestamp(data.get("file_date"), tzinfo=loc_tzinfo)
```
(note to self: d on't forget this glaring mistake + add a test for this + keep all this commented until next major version)